ext/strings.go

```diff
@@ -251,15 +257,16 @@ const (
 //	'TacoCat'.upperAscii()      // returns 'TACOCAT'
 //	'TacoCÆt Xii'.upperAscii()  // returns 'TACOCÆT XII'
 func Strings(options ...StringsOption) cel.EnvOption {
-	s := &stringLib{}
+	s := &stringLib{version: math.MaxUint32}
```

Comment:
I didn't notice it before, but could you update all references to member functions on the `stringLib` to use `*stringLib` instead?

Reply:
Done!